recipe/meta.yaml

```diff
@@ -16,15 +17,15 @@ build:
 
 requirements:
   host:
-    - python >=3.9
+    - python {{ python_min }}
     - hatchling
     - hatch-vcs
     - pip
   run:
-    - python >=3.9
-    - numpy
+    - python {{ python_min }}
+    - numpy <=2.0
```

Comment:
This constraint doesn't make sense. You are OK with numpy 2.0.0 but not 2.0.1?

Reply:
unfortunately, No.

Firecrown has not been adapted to Numpy 2.0 yet and I had some env breaking because of that for the past few weeks.